modules/ROOT/pages/create-community.adoc

```diff
@@ -108,150 +108,39 @@ Configure the settings with one of these options:
-|AnypointApiTierLimits |*Read* |*Read Access*
-|AnypointApiTiers |*Read* |*Read Access*
-|AnypointApiVersionQueries |*Read* |*Read Access*
-|AnypointApiVersions |*Read* |*Read Access*
-|AnypointApplicationOwners |*Read* |*Read Access*
-|AnypointApplications |*Read*, *Create*, *Edit*, *Delete* |*Read Access*, *Edit Access*
-|AnypointAssetCategories |*Read* |*Read Access*
-|AnypointAssetCustomFields |*Read* |*Read Access*
-|AnypointAssets |*Read* |*Read Access*
-|AnypointAssetVersions |*Read* |*Read Access*
-|AnypointCategories |*Read* |*Read Access*
-|AnypointClientProviders |*Read* |*Read Access*
-|AnypointContracts |*Read*, *Create*, *Edit*, *Delete* |*Read Access*, *Edit Access*
-|AnypointCustomFields |*Read* |*Read Access*
-|AnypointEnvironments |*Read* |*Read Access*
-|AnypointGrantTypes |*Read* |*Read Access*
-|AnypointOrganizations |*Read* |*Read Access*
-|Cases |*Read* |*Read Access*
-|CommunityApis |*Read* |*Read Access*
-|CommunityAssetVersions |*Read* |*Read Access*
-|Documents |*Read* |*Read Access*
-|===
-
-=== Configure Guest User Profile Permissions
-
-To configure permissions for the guest user profile:
-
-. Open *Setup* > *User Management Settings* and ensure that *Enhanced Profile User Interface* is enabled.
-. In the API Community Manager control panel, open *Community Builder* and click *Settings* > *General*.
-. In the *Guest user profile* section, click *_Site Name_ Profile*.
-. Click *Edit*.
-. In *Object Settings*, edit the object permissions to match those shown in the <<guest-user-perms>> table.
-. Click *Save*.
-. Return to the profile overview.
-. Click the link for *Apex Class Access*.
-. Click *Edit*.
-. Select these classes in the list and click *Add* to make them accessible:
-.. *acm_pkg.AcmSearch*
-.. *acm_pkg.AnypointApiAnalyticsController*
-.. *acm_pkg.ApiAccessRequester*
-.. *acm_pkg.ApiPortalController*
-.. *acm_pkg.ApiVersionHeader*
-.. *acm_pkg.CustomFieldsViewerController*
-.. *acm_pkg.FineGrainVisibility*
-.. *acm_pkg.LightningForgotPasswordController*
-.. *acm_pkg.LightningLoginFormController*
-.. *acm_pkg.LightningSelfRegisterController*
-.. *acm_pkg.QueryApiController*
-. Click *Save*.
+For more information about using permission sets, see https://help.salesforce.com/s/articleView?id=sf.perm_sets_overview.htm&type=5[Permission Sets].
+
+=== Configure Member User Permission Set 
+
+The permission for the member user must be assigned to the *ACM Member User* permission set so your members can access APIs, create client applications, and manage their details. 
+
+As part of the API Community Manager package, the member user is assigned automatically. After the package is installed, a trigger searches for users with a Customer Community Profile and assigns the member users to the *ACM Member User* permission set.  
+
+If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```

Comment:
```suggestion
If you want to create an new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```
```suggestion
To create a new user, you must use the API Community Diagnostics or manually set the member user profile to the *ACM Member User* permission set.
```

Reply:
Ignore the first of the two comments. I don't know why git is doubling my comments. The second comment is the correct one.

Reply:
Ignore the first of the two comments. I don't know why git made two comments instead of one. The second comment is the correct one.